circuits/core/src/algorithms/pedersen/mod.rs

```diff
@@ -39,6 +39,7 @@ pub type Pedersen1024<E> = Pedersen<E, 8, 128>;
 
 /// Pedersen is a collision-resistant hash function that takes a variable-length input.
 /// The Pedersen hash function does *not* behave like a random oracle, see Poseidon for one.
+#[derive(Clone)]
```

Comment:
sanity check: Why does `Pedersen` need `Clone`?

Reply:
good catch - this was left over from an earlier iteration of adding hashers to the execution environment. doesn't actually belong there and will delete it